libs/commands/src/interfaces/command.interface.ts

```diff
@@ -1,10 +1,7 @@
 /**
- * TODO [https://www.pivotaltracker.com/story/show/184094873]
- * Every command should have a property called `aggregateCompositeIdentifier`,
- * which holds the composite identifier of the aggregate affected by the command.
- * In the case of an index-scoped command (e.g. CREATE_X), this will be
- * generated prior to command execution. In the case of a detail-scoped command,
- * it will be the composite identifier of the aggregate on whose view the command
- * was an `action`.
+ * As a matter of our specific implementation, we have decided that every command
+ * will have an `aggregateCompositeIdentifier` property to track the aggregate
+ * ID. We introduce this to our domain by extending the following in the
+ * `api-interfaces` layer.
  */
```

Comment:
What is the point of this empty interface? It seems like the CQRS package on which our command package was based must have provided the client a way to 'inject' their base command interface. Can we do that here as well?